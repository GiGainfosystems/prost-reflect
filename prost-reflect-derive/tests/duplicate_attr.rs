use prost_reflect_derive::ReflectMessage;

#[derive(ReflectMessage)]
#[prost_reflect(file_descriptor = "FILE_DESCRIPTOR", message_name = "msg")]
#[prost_reflect(file_descriptor = "FILE_DESCRIPTOR", message_name = "msg")]
pub struct MyMessage {}

fn main() {}
