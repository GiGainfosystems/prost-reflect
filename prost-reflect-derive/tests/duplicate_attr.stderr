error: multiple #[prost_reflect] attributes
 --> tests/duplicate_attr.rs:5:1
  |
5 | #[prost_reflect(file_descriptor = "FILE_DESCRIPTOR", message_name = "msg")]
  | ^
